src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -37,9 +41,44 @@ public static JsonSerializerSettings GetSerializerSettings()
             return settings;
         }
 
+        /// <summary>
+        /// Customises the given serializer settings
+        /// using provider configuration.
+        /// Can be used by any provider, allowing the users to use a standard set of configuration attributes.
+        /// </summary>
+        /// <param name="set"></param>
+        /// <param name="config"></param>
+        /// <returns><see cref="JsonSerializerSettings" /></returns>
+        public static JsonSerializerSettings UpdateSerializerSettings(JsonSerializerSettings set, IProviderConfiguration config)
```

Comment:
Update can be applied to ANY settings, not just to the default one's stored in this class.